template/YYYY-MM-threat-model-template.md

```diff
@@ -0,0 +1,63 @@
+# Title
+
+- **Author**: Your name (@YourGitHubUserName)
+
+## Overview
+
+<!--
+Provide a succinct high-level description of the system, feature, or component being analyzed. Explain why a threat model is being created for this system, feature, or component. This section should be one to three paragraphs long and understandable by someone outside the Radius team.
+-->
+
+## Terms and Definitions
+
+<!--
+Include any terms, definitions, or acronyms that are used in this threat model document to assist the reader. They may or may not be part of the user-facing experience once implemented, and can be specific to this design context.
+-->
+
+## System Description
+
+<!--
+Provide a detailed description of the system or feature being modeled. Include information key components, and interactions with other systems.
+-->
+
+### Architecture
+
+<!-- Overview of the system architecture of the component that is being discussed in this document. -->
+
+### Clients
+
+<!-- Clients that communicate with the component that is being reviewed in the threat model. -->
+
+## Trust Boundaries
+
+## Data Flows
+
+<!--
+Include a diagram of the system architecture, showing how different components interact. Highlight any areas where security controls are implemented or where threats might be present.
+-->
+
+### Diagram
+
+<!-- The diagram for the threat model. It can be done by using Microsoft Threat Modeling Tool. -->
+
```

Comment:
Add a section for security-sensitive implementation details. This is a series of yes/no questions. 

- Does the code implement or use cryptography?
- Does the code store secrets?
- Docs the code parse any custom format or handle untrusted data?